spec/factory_specs/fois_spec.rb

```diff
@@ -127,7 +127,7 @@
         expect(managing_assignment.state).to eq 'accepted'
         expect(managing_assignment.team).to eq team_disclosure_bmt
 
-        expect(kase.transitions.size).to eq 0
+        expect(kase.transitions.size).to eq 1
```

Comment:
Did Alex know that this change would affect ALL case types, and not just Offender SARs? I think it's great, but just checking... :)

Reply:
@Polsonby  please check my comment for your similar feedback for another change